### DISCUSSION TITLE
Onboarding new room leaders

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#169] [Coffees]
@Virtual-Coffee/coffee-talk-team we're going to be onboarding some new room leaders and I'd like to get some docs going. I wanted to discuss it here before moving it to an issue and PR. 

Here's a very rough sketch of the process:

- Add to team and ask them to review docs
- Pair with an experienced room leader. 
- The first time (or more, depending on their comfort), the experienced leader will do the leading.
- Once new RL is comfortable, they'll go to the next stage: intros in the breakout rooms. An experienced leader will be responsible for facilitating the conversation.
- When new RL is ready to lead on their own, they let us know. They'll be paired again with an experienced RL, who will be there for support and to offer feedback after coffee. (Maybe we designate mentor RLs?)

What should we add or change?

**Comment by tkshill** [comment]
This process seems great! I really like the idea of the first role of new RLs is to review and give feedback to current docs. If we add the bulk of the new folks this week, are we comfortable with giving them the week to do the review, and start pairing them during next week's coffees? At least as observers?

**Reply by BekahHW** [reply]
I am if they're comfortable with it.

**Comment by nickytonline** [comment]
I think this is a great way to get new RLs in to the flow. 🥳

**Comment by tkshill** [comment]
I can get this into an issue/PR but not until the weekend.

**Comment by meg-gutshall** [comment]
💯
